src/main/java/RunceDemoClass.java

```diff
@@ -0,0 +1,31 @@
+/*
+pspspspsps
+ */
+public class RunceDemoClass implements Tradable, Drivable{
+    private int speed;
+
+    @Override
+    public int getPrice() {
+        return 0;
```

Comment:
You can probably add an instance variable for price, does it make sense to always have price at 0?